views/results/_form.html

```diff
@@ -1,8 +1,13 @@
 <div class="card card-upload">
-    <form class="form form--rounded-input form-upload" action="/results" method="post" enctype="multipart/form-data">
-        <div class="form__input-group">
-            <input id="file" name="file" type="file" accept="text/csv" class="form__input" required>
-        </div>
+    <h3 class="card__title">Upload your keywords here</h3>
+    <p class="card__subtitle">
+        {{ render_icon "info" "icon--sm" }}
+        Only CSV file allowed, check
+        <a href="/files/sample.csv"> this sample file </a>
```

Comment:
Dont we need to add `sample.csv`?

Reply:
Nice catch! I forgot that I ignore all the files in the `static` folder 😅 
added in [baaaa54](https://github.com/carryall/go-google-scraper-challenge/pull/57/commits/baaaa54a65787501c122a8678d8bb5f91b83c123)